pkg/features/features.go

```diff
@@ -221,7 +221,9 @@ var UDPRouteFeatures = sets.New(
 
 const (
 	// This option indicates general support for service mesh
-	SupportMesh SupportedFeature = "Mesh"
+	SupportMesh                  SupportedFeature = "Mesh"
+	SupportMeshClusterIPMatching SupportedFeature = "MeshClusterIPMatching"
+	SupportMeshConsumerRoute     SupportedFeature = "MeshConsumerRoute"
```

Comment:
We should define a new set of consts for mesh extended features, to be coherent with the other features in this file. Also, since these are public symbols, we should provide a comment.